content/v2.0/security/https_setup.md

```diff
@@ -0,0 +1,145 @@
+<!-- as well as certificates that combine the private key file and the signed certificate file into a single bundled file (`.pem`). -->
+
+The following two sections outline how to set up HTTPS with InfluxDB
+[using a CA-signed certificate](#set-up-https-with-a-ca-signed-certificate)
+and [using a self-signed certificate](#set-up-https-with-a-self-signed-certificate).
+
+## Set up HTTPS with a CA-signed certificate
+
+1. **Install the certificate**
+
+    Place the private key file (`.key`) and the signed certificate file (`.crt`) in the `/etc/ssl/` directory.
+
+2. **Set certificate file permissions**
+
+    Users running InfluxDB must have read permissions on the TLS certificate.
+
+    {{% note %}}You may opt to set up multiple users, groups, and permissions.
+    Ultimately, make sure all users running InfluxDB have read permissions for the TLS certificate.
+    {{% /note %}}
+
+    Run the following command to give InfluxDB read and write permissions on the certificate files.
+
+    ```bash
+    sudo chmod 644 /etc/ssl/<CA-certificate-file>
+    sudo chmod 600 /etc/ssl/<private-key-file>
+    ```
+
+3. **Run `influxd` with TLS flags**
+
+    Start InfluxDB with TLS command line flags:
+
+    ```bash
+    influxd \
+    --tls-cert "/etc/ssl/influxdb-selfsigned.crt"\
+    --tls-key "/etc/ssl/influxdb-selfsigned.key"
+    ```
+
+That's it! You've successfully set up HTTPS with InfluxDB.
+
+## Set up HTTPS with a self-signed certificate
+
+1. **Generate a self-signed certificate**
+
+    The following command generates a private key file (`.key`) and a self-signed
+    certificate file (`.crt`) which remain valid for the specified `NUMBER_OF_DAYS`.
+    It outputs those files to `/etc/ssl/` and gives them the required permissions.
+
+    ```bash
+    sudo openssl req -x509 -nodes -newkey rsa:2048 \
+    -keyout /etc/ssl/influxdb-selfsigned.key \
+    -out /etc/ssl/influxdb-selfsigned.crt \
+    -days <NUMBER_OF_DAYS>
+    ```
+
+    When you execute the command, it will prompt you for more information.
+    You can choose to fill out that information or leave it blank; both actions generate valid certificate files.
+
+2. **Run `influxd` with TLS flags**
+
+    Start InfluxDB with TLS command line flags:
+
+    ```bash
+    influxd \
+    --tls-cert "/etc/ssl/influxdb-selfsigned.crt" \
+    --tls-key "/etc/ssl/influxdb-selfsigned.key"
+    ```
+
+That's it! You've successfully set up HTTPS with InfluxDB.
+
+## Connect Telegraf to a secured InfluxDB instance
+
+Connecting [Telegraf](/telegraf/latest/) to an InfluxDB instance that's using
+HTTPS requires some additional steps.
+
+In the Telegraf configuration file (`/etc/telegraf/telegraf.conf`), edit the `urls`
+setting to indicate `https` instead of `http`.
+(Change `localhost` to the relevant domain name if necessary.)
+If you're using a self-signed certificate, uncomment the `insecure_skip_verify` setting and set it to `true`.
```

Comment:
To connect [Telegraf](/telegraf/latest/) to an InfluxDB 2.0 instance with TLS enabled, update the following `influxdb_v2` output settings in your Telegraf configuration file:

- Update `urls` to use `https`.
- If using a self-signed certificate, uncomment and set `insecure_skip_verify` to `true`.